certified-connectors/InEight/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3049 @@
+        "operationId": "Users_Import",
+        "parameters": [
+          {
+            "name": "X-IN8-TENANT-PREFIX",
+            "in": "header",
+            "description": "The tenant-prefix of your Project Suite base URL. For example, if you are accessing Project Suite at *https://domain.hds.ineight.com/* then the value for this parameter would be **domain**",
+            "x-ms-summary": "The tenant-prefix of https://Domain.hds.ineight.com/ would be \"Domain\"",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "Ocp-Apim-Subscription-Key",
+            "in": "header",
+            "description": "This is found in in your APIM Profile.",
+            "x-ms-summary": "This is found in in your APIM Profile.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "required": [
+                  "FirstName",
+                  "LastName",
+                  "EmailAddress"
+                ],
+                "properties": {
+                  "FirstName": {
+                    "type": "string",
+                    "description": "FirstName"
+                  },
+                  "LastName": {
+                    "type": "string",
+                    "description": "LastName"
+                  },
+                  "IsActive": {
+                    "type": "boolean",
+                    "description": "IsActive"
+                  },
+                  "Language": {
+                    "type": "string",
+                    "description": "Language"
+                  },
+                  "DecimalFormat": {
+                    "type": "string",
+                    "description": "DecimalFormat"
+                  },
+                  "DateFormat": {
+                    "type": "string",
+                    "description": "DateFormat"
+                  },
+                  "EmployeeId": {
+                    "type": "string",
+                    "description": "EmployeeId"
+                  },
+                  "VendorCode": {
+                    "type": "string",
+                    "description": "VendorCode"
+                  },
+                  "SourceSystemName": {
+                    "type": "string",
+                    "description": "SourceSystemName"
+                  },
+                  "SourceSystemId": {
+                    "type": "string",
+                    "description": "SourceSystemId"
```

Comment:
@InEightDevOps All the description values of properties are expected to be free from grammatical and spell errors. Please provide valid description values.

Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)